src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -4,11 +4,11 @@
 import java.time.format.DateTimeFormatter;
 
 public class SalaryInfo {
-    static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
     static final int DATE_INDEX = 0;
     static final int NAME_INDEX = 1;
     static final int HOURSPOSITION = 2;
     static final int SALARYPOSITION = 3;
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```
what about all other constants?